solr/core/src/java/org/apache/solr/handler/RequestHandlerBase.java

```diff
@@ -212,6 +216,14 @@ public abstract void handleRequestBody(SolrQueryRequest req, SolrQueryResponse r
 
   @Override
   public void handleRequest(SolrQueryRequest req, SolrQueryResponse rsp) {
+    ThreadStats cpuStats = null;
+    if (publishCpuTime == null) {
+      publishCpuTime = Boolean.getBoolean(ThreadStats.ENABLE_CPU_TIME);
+    }
```

Comment:
Can't this be added to the field's declaration instead of here?  And would then be a primitive `boolean`

Reply:
If it's then protected, could be accessed by SearchHandler, removing the field there.